README.md

```diff
@@ -28,8 +28,7 @@ Other installation methods are available, see [documentation](https://moira.read
 To build and run tests, first get all dependencies:
 
 ```bash
-go get github.com/kardianos/govendor
-govendor sync
+go get ./...
```

Comment:
Also this.

Reply:
Readme was rewritten to fit go modules practices